src/modules/app/createReduxStore.js

```diff
@@ -51,6 +53,7 @@ const createReduxStore = (
     darkMode: false,
 
     cities: defaultCitiesState,
+    contentLanguage: DEFAULT_LANGUAGE,
```

Comment:
Probably would be cleaner if a defaultContentLanguage should be introduced in the StateType like defaultCitietsState and set it to DEFAULT_LANGUAGE.